### PR TITLE
QuinticWalk: added stability criteria for imu angular velocity

### PR DESCRIPTION
**Pull request by SammyRamone**
tested in visualization, good values still have to be found. since imu is default deactivated currently, I think we can merge this on the master

### CODE REVIEW

File: bitbots_quintic_walk/src/QuinticWalkingNode.cpp [hunk 1]
```diff
@@ -230,10 +230,14 @@ void QuinticWalkingNode::imuCb(const sensor_msgs::Imu msg) {
...
-
         pitch = pitch - wanted_pitch;
 
-        if (abs(roll) > _imu_roll_threshold || abs(pitch) > _imu_pitch_threshold) {
+        // get angular velocities
+        double roll_vel = msg.angular_velocity.x;
+        double pitch_vel = msg.angular_velocity.y;
+
+
+        if (abs(roll) > _imu_roll_threshold || abs(pitch) > _imu_pitch_threshold || pitch_vel > _imu_pitch_vel_threshold || roll_vel > _imu_roll_vel_threshold) {
```
Comment (timonegk) [inline bitbots_quintic_walk/src/QuinticWalkingNode.cpp]: I think that the absolute value of the velocities should be used

Comment (SammyRamone) [reply bitbots_quintic_walk/src/QuinticWalkingNode.cpp]: good point. I corrected it :)

**Comment by timonegk** [comment]
Ping @Flova

**Comment by Flova** [comment]
Looks good.
A nice feature would be to use conditions similar to the falling conditions.
The falling detector only triggers if rotation and orientation are in the same direction. Also the rotation velocity theshold gets dynamicly lower the closer you get to the orientation threshold.

**Pull request merged**